Gemfile

```diff
@@ -0,0 +1,22 @@
+source 'https://rubygems.org'
+
+ruby '2.4.0'
```

Comment:
Sorry, what do you mean? 
Do you mean bundle lock? bundle lock doesn't make anything for me.